queries.sql

```diff
@@ -1,11 +1,53 @@
 /*Queries that provide answers to the questions from all projects.*/
-
-
 SELECT * FROM animals WHERE name LIKE '%mon';
 SELECT name FROM animals WHERE date_of_birth >='2016-01-01' AND date_of_birth <= '2019-12-31';
 SELECT name from animals WHERE neutered = 'true' AND escape_attempts <3;
 SELECT date_of_birth FROM animals WHERE name ='Agumon' OR name ='Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 'true';
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg >=10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >=10.4 AND weight_kg <= 17.3;
+
+-- day 2 tasks2 query 1
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM ANIMALS;
+ROLLBACK;
+
+-- tasks2 query 2
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM ANIMALS;
+-- take screenshort 1
+
+-- tasks2 query 3
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] You have to add this query `SELECT * FROM ANIMALS;` before also as after deleting we have to check whether the table is deleted and then after rollback we will check if its again restored.